apps/parks-app/src/bin/pgdb.js

```
@@ -0,0 +1,106 @@
+  pg(query, function(err, rows, result) {
+    if(err) {
+      return console.error(error_response, err);
+    }
+    var response = 'Data import completed!';
+    return response;
+  });
+};
+
+function insertMapPinSQL(pin) {
+  var query = '';
+  var escape = /'/g
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)